charts/victoria-metrics-distributed/README.md

```diff
@@ -0,0 +1,209 @@
+# Victoria Metrics Helm Chart for Setting up VMCluster on Multiple Availability Zones
+
+![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)  ![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square)
+[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/victoriametrics)](https://artifacthub.io/packages/helm/victoriametrics/victoria-metrics-distributed)
+[![Slack](https://img.shields.io/badge/join%20slack-%23victoriametrics-brightgreen.svg)](https://slack.victoriametrics.com/)
+
+A Helm chart for Running VMCluster on Multiple Availability Zones
+
+## Prerequisites
+
+* Install the follow packages: ``git``, ``kubectl``, ``helm``, ``helm-docs``. See this [tutorial](../../REQUIREMENTS.md).
+
+* PV support on underlying infrastructure.
+
+* Multiple availability zones.
+
+## Chart Details
+
+This chart sets up multiple VictoriaMetrics cluster instances on multiple [availability zones](https://kubernetes.io/docs/setup/best-practices/multiple-zones/), provides both global write and read entrypoints.
+
+The default setup is as shown below:
+
+![victoriametrics-distributed-topology](./victoriametrics-distributed-topology.png)
+
+For write:
+1. extra-vmagent(optional): scrapes external targets and all the components installed by this chart, sends data to global write entrypoint.
+2. vmauth-global-write: global write entrypoint, proxies requests to one of the zone `vmagent` with `least_loaded` policy.
+3. vmagent(per-zone): remote writes data to availability zones that enabled `.Values.availabilityZones.allowIngest`.
```

Comment:
we need to mention that it acts as a buffer in the same time, for restoring data consistency in case if any zone is unavailable. Hence, we need to provide recommendations regarding disk space capacity.